capi/VideoDecoderCapi.h

```diff
@@ -20,8 +20,8 @@
 #ifndef __ENABLE_CAPI__
 #define __ENABLE_CAPI__ 1
 #endif
-#include "interface/VideoDecoderDefs.h"
-#include "interface/VideoCommonDefs.h"
+#include "VideoDecoderDefs.h"
+#include "VideoCommonDefs.h"
```

Comment:
Hi @chivakker 
Maybe we should remove capi/xxx.h in yami-utils. yami-utils need only use installed header from libyami.